keyboards/ai03/lily/keyboard.json

```diff
@@ -0,0 +1,189 @@
+{
+    "manufacturer": "ai03 Design Studio",
+    "keyboard_name": "Lily",
+    "maintainer": "ai03",
+    "bootloader": "atmel-dfu",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "B0",
+        "on_state": 0
+    },
+    "matrix_pins": {
+        "cols": ["F0", "F1", "F4", "F5", "F6", "F7", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4"],
+        "rows": ["B2", "B7", "D5", "B1", "D3"]
+    },
+    "processor": "atmega32u4",
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```

Comment:
May not need this.
```suggestion
```

Reply:
Used in testing phase to quickly check functionality or RGB single colors. It's faster to set the lighting to that to quickly check.